src/auto/file_i_o_stream.rs

```diff
@@ -1,15 +1,15 @@
-// Copyright 2013-2017, The Gtk-rs Project Developers.
-// See the COPYRIGHT file at the top-level directory of this distribution.
-// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+// This file was generated by gir (https://github.com/gtk-rs/gir @ bd67955)
+// from gir-files (https://github.com/gtk-rs/gir-files @ db49619)
+// DO NOT EDIT
```

Comment:
Cool changes 😉

Reply:
Yes, boring :(

Reply:
Would be sweet if this info could be output in a separate file instead, with the option to version control it (or not) left to the repo maintainer